apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -506,14 +610,7 @@ export const getSearchResults = async ({
   const queriedResources = getResourcesWithLastUpdatedAt({
     siteId: Number(siteId),
   })
-    .where("Resource.type", "in", [
-      // only show user-viewable resources (excluding root page, folder meta etc.)
-      ResourceType.Page,
-      ResourceType.Folder,
-      ResourceType.Collection,
-      ResourceType.CollectionLink,
-      ResourceType.CollectionPage,
-    ])
+    .where("Resource.type", "in", resourceTypes)
```

Comment:
didn't check, but a friendly reminder to update the callsites to pass in `resourceTypes`